docs/canvas/canvas-edit-workpads.asciidoc

```diff
@@ -25,12 +25,12 @@ For example, to change the index pattern for a set of charts:
 Specify the variable options.
 
 [role="screenshot"]
-image::images/specify_variable_syntax.png[Specify the variable syntax]
+image::images/specify_variable_syntax.png[Image describing how to specify the variable syntax]
```

Comment:
It would be great if the image descriptions would describe the images and not just what they are for. 

As it stands, when reading these docs, a person using a screen reader may hear something along the lines of this: "Heading, specify the variable options. Image, copy the variable syntax. Copy the variable, then apply it to each element you want to..."

That's a simplified example of how a screen reader may go through the page but I think it's enough to see two things:

1. The screen reader will identify the type of content for us so, like @timroes said, we don't need to say that something is an image
2. There's a lot of info lost by not explaining what the actual image is and just explaining what the image is for. Good alt text will explain the actual contents of the image (at least, the relevant bits).

As an example, a good description for this image might be something like this:
```suggestion
image::images/specify_variable_syntax.png[Screenshot of a form titled "Add variable". A dropdown labelled "Type" is set to "String". A text field labelled "Name" is filled in with "indexName". A text field labelled "Value" is filled in with "my_new_index_name".]
```

I'm not 100% sure what the important parts of the screenshot were so I assumed the whole thing was except the save/cancel buttons but you could use your own discretion on whether on not the filled in values, for example, are of any importance.